ChangeLog

```diff
@@ -15,6 +15,8 @@ ver. 0.9.5 (2016/XX/XXX) - wanna-be-released
      - failregex of previous monit version merged as single expression.
    * filter.d/postfix.conf, filter.d/postfix-sasl.conf
      - extended failregex daemon part, matching also `postfix/smtps/smtpd` now (gh-1391)
+   * grave tags substitution bug fixed: wrong recursion recognition by some constellation 
+     within substituteRecursiveTags (gh-1417). Affected actions like `bsd-ipfw`, etc.
```

Comment:
Such changelog describes the symptom which was fixed, which is indeed also useful (for users), but not the change which was done (useful for precious us).  Also not sure what "by some constellation" means.  What about
```
* fixed a bug within tags substitutions because of incorrect detection of recursion in case of multiple substitutions
  (affected actions: `bsd-ipfw`). Now also tracks the actual content of the already substituted tags
```